auth_proxy/proxy/flask.py

```diff
@@ -99,3 +105,32 @@ def _patient_security(self):
             return 'Patient/{}'.format(patient)
         except AttributeError:
             return 'public'
+
+
+class OpenFlaskClient(FlaskClient):
+    """Subclass of FlaskClient, overridden to prevent any security checks """
+    def __init__(self, url, orig):
+        super().__init__(url, orig)
+
+    def request(self):
```

Comment:
This works, but I think it obscures the intention of this class. OpenFlaskClient doesn't exist to do requests differently, it exists to disable security. To that effect, I think it makes more sense to override the `_scope_security` and `_patient_security` methods instead. You'd have to change their interface though. Something like changing the original `request` method to

```
if len(path) == 1:
    self._scope_security(args)
    self._patient_security(args)
```

and `_scope_security` to:

```
def _scope_security(self, args):
    """ Determine which categories the client should be allowed to see
    based on their approved scopes.
    """
    try:
        labels = ','.join(['public'] + self.orig.oauth.access_token.security_labels)
    except AttributeError:
        labels = 'public'
    args.append(('_security', labels))
```

Then, OpenFlaskClient can just define `_scope_security` without needing to reimplement request.

```
def _scope_security(self, args):
    pass
```

Reply:
I agree. I also did the same for when the "_security" tag gets stripped, this way you could potentially test out the FHIR level security without the proxy level security getting in the way.